utils/python/CIME/case_st_archive.py

```diff
@@ -180,7 +181,7 @@ def get_histfiles_for_restarts(case, archive, archive_entry, restfile):
         cmd = "ncdump -v %s %s " %(rest_hist_varname, os.path.join(rundir, restfile))
         rc, out, error = run_cmd(cmd)
```

Comment:
Yes - we do not want that reported each time - since its just to determine if that variable happens to exist and it was checking this on all components and files. It was very misleading to have it there in the first place. I have replace this with a much more extensive summary of the files that are being short term archived.